crates/sui-transaction-builder/src/lib.rs

```diff
@@ -97,6 +97,79 @@ impl TransactionBuilder {
         }
     }
 
+    /// Construct the transaction data for a dry run (uses min gas budget)
```

Comment:
I'm not sure what the "uses min gas budget" part means here, because this function is being passed a gas budget.

Reply:
Ah, forgot to remove this oudated comment.

Reply:
This comment still seems to be around, but was marked as resolved -- are there changes that need to be pushed still?